### PR TITLE
File system: translation + unused anchors removed

### PR DESCRIPTION
**Pull request by innabelaya**
@tadatuta

### CODE REVIEW

**Comment by tadatuta** [comment]
branch should be rebased

**Comment by innabelaya** [comment]
@tadatuta rebased

File: method/filesystem/filesystem.en.md [hunk 1]
```diff
@@ -1,339 +1,186 @@
...
-```
-myfacebook/
-  myfacebook.css
-  myfacebook.js
-  myfacebook.html
-```
+In a BEM project, the code is broken down into small independent parts, to make working with individual blocks easier. Before they are sent to the browser, the files are assembled and optimized. That way we separate the human-manipulated code from the code that is sent to the browser.
 
-Files `myfacebook.css` and `myfacebook.js` contain all the CSS/JS codebase of the project and are linked from HTML as is.
+In the file system, the codebase of any BEM project is organized according to the following principles:
```
Comment (tadatuta) [inline method/filesystem/filesystem.en.md]: let's get rid of `any` here?

File: method/filesystem/filesystem.en.md [hunk 2]
```diff
@@ -1,339 +1,186 @@
...
 
-There is now process of building and optimizing such a files. It is the simplest case to start.
+- [A block implementation is divided into separate parts](#a-block-implementation-is-divided-into-separate-files)
+- [Optional elements and modifiers are stored in separate files](#optional-elements-and-modifiers-are-stored-in-separate-files)
+- [Files are grouped per meaning and not per type](#files-are-grouped-per-meaning-and-not-per-type)
```
Comment (tadatuta) [inline method/filesystem/filesystem.en.md]: per -> by ? (I'm not sure actually)

File: method/filesystem/filesystem.en.md [hunk 3]
```diff
@@ -1,339 +1,186 @@
...
-    * File name coincides with the name of the block
-    * Technical implementation of the block is placed in the files with the appropriate extensions
-  * An element corresponds to a file `block-name__element-name`
-    * File name coincides with the name of the element
-    * Technical implementation of the element is placed in the files with the appropriate extensions
-  * A modifier corresponds to a file `block-name_name-of-the-modifier_value-of-the-modifier`
-    * The name of the file consists of the name of the modifier along with it's value
-    * Technical implementation of the modifier is placed in the files within this directory
+- **Only necessary blocks are included in the project**
+  Blocks are implemented as independent enteties. This enables us to configure a build in a way that ensures that only relevant blocks are included in the project.
```
Comment (tadatuta) [inline method/filesystem/filesystem.en.md]: enteties -> entities

**Comment by tadatuta** [comment]
I'm done

File: method/filesystem/filesystem.en.md (hunk 2 above)
Comment (innabelaya) [inline method/filesystem/filesystem.en.md]: by is correct, thank you

**Comment by innabelaya** [comment]
@tadatuta corrected, thank you

**Pull request merged**